pysph/tools/ipython.py

```diff
@@ -0,0 +1,641 @@
+class Viewer(object):
+
+    '''A class for the pysph jupyter notebook Viewer2D.
+
+    Attributes
+    ----------
+
+        path              String containing the path of the output directory
+        paths_list        List of paths to the output files
+        cache             Cached data
+
+    Methods
```

Comment:
Do you really need the attributes and methods listed.  Instead provide an example of how to use this.